### PR TITLE
Added getValue(), inputValue() methods in Label Class. Made Changes t…

### PR DESCRIPTION
**Pull request by Lukasz-Kawa**
…o the AccountAddressPage, CheckoutPage, HandbagsCataloguePage, MyAccountPage, ReviewOrderPage Classes. Added asserJ and surfire dependencies in pom.xml. Added AccountTests, CartTests, ChangeAddressTests classes.

### CODE REVIEW

File: src/main/java/tests/CartTests.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        ReviewOrderPage reviewOrderPage = new ReviewOrderPage(driver);
+        reviewOrderPage
+                .chooseProceedCheckout();
+
+        CheckoutPage checkoutPage = new CheckoutPage(driver);
+        String moneyOrderAddress = checkoutPage
+                .getMoneyOrderAddress().replaceAll("\n", ",");
+
+        assertThat(moneyOrderAddress)
+                .contains("Vintage Bags")
```
Comment (katarzynaboguslawska) [inline src/main/java/tests/CartTests.java]: you could have used the Payment model here

File: src/main/java/tests/CartTests.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                .addBagToCart()
+                .proceedToCheckout();
+
+        ReviewOrderPage reviewOrderPage = new ReviewOrderPage(driver);
+        reviewOrderPage
+                .removeItemFromCart();
+        assertNotEquals("Url is the same", "http://demo.shopizer.com:8080/shop/", this.driver.getCurrentUrl());
+    }
+
+    @Test //TODO fix problem with sending of the quantity
```
Comment (katarzynaboguslawska) [inline src/main/java/tests/CartTests.java]: fix or remove TODO ;P

File: src/main/java/tests/CartTests.java (hunk 1 above)
Comment (Lukasz-Kawa) [reply src/main/java/tests/CartTests.java]: Will work on that later on as i struggled to make an assertion.

File: pom.xml [hunk 3]
```diff
@@ -35,8 +35,27 @@
...
+        </dependency>
+
 
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
```
Comment (katarzynaboguslawska) [inline pom.xml]: nice! possible to run from cmd ;)

**Pull request merged**